src/main/groovy/life/qbic/datamodel/dtos/imaging/properties/Detector.groovy

```diff
@@ -12,6 +12,7 @@ package life.qbic.datamodel.dtos.imaging.properties
  * @author Sven Fillinger
  * @since 1.10.0
  */
+
```

Comment:
I am unsure why this is displayed empty. My IDE shows the code that you implemented here @sven1103